website/docs/concepts/recorder_service/README.md

```diff
@@ -22,10 +22,12 @@ first traverse the field, recording data to train a model to detect weeds.
 The Recorder Service is a subscriber to the following services:
 
 - Canbus
+- Filter
+- GPS
 - Oak (Multiple Oak topics may be available, depending on the number of Oak cameras connected,
 e.g., oak0, oak1, ...)
```

Comment:
We should start making these changes to get people used to it
```suggestion
e.g., oak/0, oak/1, ...)
```

Reply:
Agreed. Implemented here 49f2fd94e20a76c958d302609e91d8c1a3eec936